### ISSUE TITLE
IScroll can't  smoothly scroll in Android broswer

### ISSUE THREAD

**Issue by loveLibra** [cubiq/iscroll#966]
I test many versions of iscroll5, and I found when the version >= 5.0.7, It didn't perform good in android broswer, why?

**Comment by focusqfb** [comment]
我好像也遇到这个问题，android5.0以上比较卡....

**Comment by kaansoral** [comment]
Usually extremely small things make major changes like these

Is this the latest version that performs well for you: https://github.com/cubiq/iscroll/blob/v5.0.6/build/iscroll.js

Also which android version are you testing on?

From my experience, android performs a bit better than iOS, with the latest version

**Comment by focusqfb** [comment]
thanks for you help ,have a nice day!

**Comment by loveLibra** [comment]
@kaansoral I am using iscroll-probe, I think it's the main reason...maybe iscroll performs good

**Comment by petemolinero** [comment]
This is true for me too; using iScroll-probe on android is super laggy for me, basically unusable.

**Comment by edgardz** [comment]
Same here. It just scrolls a little bit, then I have to swipe again to scroll another little bit...
Update: Tested with the repo version (v5.2.0-snapshot), and it seems to work well.

**Comment by darior87** [comment]
neither on v5.2.0 works well

**Comment by Fental** [comment]
Try CSS `touch-action: none;` on wrapper DOM
